cpp/ql/src/Critical/MissingNullTest.ql

```diff
@@ -5,6 +5,7 @@
  * @id cpp/missing-null-test
  * @problem.severity recommendation
  * @tags reliability
+ *       security
```

Comment:
I know that dereferencing null is UB, but how often does it actually cause a security problem?

Reply:
I was looking at it as a potential denial of service vector, but that argument could go either way.

Reply:
That makes sense. It has a CWE number, and tagging as `security` is be consistent with what we decided for other queries.